02_Python_Basics/Methode.appand.py

```diff
@@ -0,0 +1,108 @@
+import uuid
+import json
+
+with open('base.json', 'r') as data_file:
+    data = json.load(data_file)
+
+class Member:
+    def __init__(self, name: str, age: int, secretidentity: str, powers: list):
+        self.name = name
+        self.age = age
+        self.secretidentity = secretidentity
+        self.powers = powers
+        self.uuid = str(uuid.uuid4())
+
+    def __str__(self):
+        return f"Name: {self.name}, Age: {self.age}, Secret Identity: {self.secretidentity}, Powers: {', '.join(self.powers)}"
+
+class Squad:
+    def __init__(self, squad_name: str, home_town: str, formed: int, status: str, secret_base: str, active: bool):
+        self.squad_name = squad_name
+        self.home_town = home_town
+        self.formed = formed
+        self.status = status
+        self.secret_base = secret_base
+        self.active = active
+        self.members = []
+
+    def add_member(self, name: str, age: int, secretidentity: str, powers: list):

```

Comment:
Für die Methoden und die Klassen eine kurze Beschreibung, was sie macht (in einem String)